modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java

```diff
@@ -348,6 +347,44 @@ public void testItIsNotPossibleToCreateTablesInSystemSchema(String schema) {
                 () -> sql(format("CREATE TABLE {}.SYS_TABLE (NAME VARCHAR PRIMARY KEY, SIZE BIGINT)", schema.toLowerCase())));
     }
 
+    @ParameterizedTest
+    @MethodSource("reservedSchemaNames")
+    public void testCreateSystemSchemas(String schema) {
```

Comment:
can we make these tests as SqlLogic tests?

Reply:
But, as you correctly noted in such a case we will miss return code check

Reply:
it is so, but it doesn't look like a big problem. Otherwise, we can introduce check error codes in SqlLogic tests and move all such simple cases. Up to you